sympy/stats/crv_types.py

```diff
@@ -4625,7 +4625,7 @@ def WignerSemicircle(name, R):
     Returns
     =======
 
-    A `RandomSymbol`.
+    A RandomSymbol.
```

Comment:
An option is to change this to 
```
A :class:`~.RandomSymbol`.
```
which should provide a clickable link to the RandomSymbol class (subject to that I do not remember the exact syntax...).

But this change is clearly more correct than it was.